src/api/contract/configuration/v1.ts

```diff
@@ -13,11 +13,11 @@ export namespace ConfigurationV1 {
         readonly hostPath: string;
     }
 
-    export interface WslKubeconfigPath {
+    export interface WSLKubeconfigPath {
```

Comment:
I see that we also have in the codebase existing occurrences where WSL is not all uppercase ("getUseWsl").
Should we change them as well? Or change the existing occurrences where "WSL" is used to "Wsl"?
Anyway that's not a big deal, feel free to keep as is.

Reply:
No, don't worry about changing it elsewhere - the internal code is already all over the place in terms of style (which is why it didn't register with me before).  But in the API, the name is consumed by other people, so it's more important to conform to the language community's preferred conventions.

Which, given that there are about four million fragmented JavaScript "communities," may be a bit idealistic...

data:image/jpeg;base64,/9j/4AAQSkZJRgABAQAAAQABAAD/2wCEAAkGBxITEhUTExMWFRUXGBUYFRYYGBgYFxcWGBgXGBgYHRgYHSggGB0lHRcYITEhJSkrLi4uGB8zODMtNygtLisBCgoKDg0OFxAQFy0dHR0tLS0tLS0tLS0tLS0tLS0tKy0tLS0tLSstLS0tLS0rLS0tLS0tLS0tNy0tOC0tLSs3K//AABEIAKgBKwMBIgACEQEDEQH/xAAcAAABBQEBAQAAAAAAAAAAAAADAgQFBgcAAQj/xABBEAABAwIEAgcGBQMBCAMBAAABAgMRAAQFEiExQVEGEyJhcYGRMqGxwdHwBxQjQuEVUmLxFkNTY3KCkqIzk7Il/8QAGAEAAwEBAAAAAAAAAAAAAAAAAAECAwT/xAAhEQEBAQEAAgMBAQADAAAAAAAAARECAyESMUFREyIyYf/aAAwDAQACEQMRAD8A1zrO+hvPQkmeFICq52I12oxOoS2Mujx/0qcSvSKhbZQDk99TNBglZnelGkOECo+5uDT55NIZ0jjRGkpVqDVf/M0Vq6IMyRTyDE44wKB1CRqAAY5cOVN3MWCUyQpX/TE+81EXHStOwaJ3GqgPcAam3DnNqbwoDtESCoyfl7qc3AnThVQtekqkk9hJHiRVkwjERcIKspSQYIOvfoaJ1KV5sLQYI1pwyudj9ihqSJ3ozUAafc0EIK4KoSrgcSNppsb4ZoAnvFPBqSNRmKs7Ec4p6lVM71ySJ4Ggntokp3PLTlRnrlKRmUdK8QQQDQrphKhBnTWgy0XYVqNqI0s0xS6hECf4p3bqChIM99PASZk6mhLSqN/TfWjOKA3IoPXAqIowAuJzpUlQOgGscYmm2DqIBHOPTWpPJxmmloyO3oNDSMRwSfHelJTPE+H3vXqU12WDQHi015kE0RdCPrQHpZE0B1EGnAJoahz1+9qCAbBjXWm8EAjgFGBvxNPFmBNN7bVM8yTB8TTkAraid/OuuUhQhQBHftSm4pTtMjJWEtGZTv3n3cqdMYa0EgZRR2iKICKeBEHGXOY9KIcVWRuPSkXODiBlOvGeP0pAaDKVLdT2RA5nyrLTx5+a4wPfR/60sDXKB3zUDeYxmENoSkczqr1qJfWT7SvUiovkjXnxX9W+5xYKSCCD4c6YC9zTNVRdylB1eAmP3JGnCmT/AEjYTop8k/4kn/8ANH+0V/l/6uL1yQN4pqvFQOJV4a+/aqNddMWh7KFub6qOUe+TUe/0ofUOwEtjwzH1NTfJfw5xy0FeIPOGIhPLn4/frQ22AVEDMpU65dTPfrp51ndtcvOkZ1qV3EmPTati6L4an8s2qCCpM8R5+e9Obfsur8Z6NrLB5Mubf2g6nxPDynxqwtYgG0hKWwlI2A2pTWHAalSj3aUQ2yRw95rWSRhbaAcW27Hv/iknGztk9/8AFerYnaBzoarU6xr30EQu+CyTl7vaP0o7N2lIkJBI76atolQTsTptTkYUrWV+gNVNBa8XPCB5/wAUJ7FQoQQO/U/ShXGFRqD/ABTd/DkoSVOLCQBqZgDx0paeJAYrEAJEeNenFT/b76p7uOMSQ2ouAbqA9wBIknXXYUwR0mc3KUpTpI1Om8Az2iBuRoCQNaMP41d3LpJMlH/t/FFZxfKIS3/7fxUdg1226mCkBQgkTrB8+enpzqVTZtnhHmaNow1u7/MJy68kq1+FMGMfKdcnqqfkKc3bCULIGug7/Km7eD5wVZso4aAyeO0VUv8ASwU9KDwQPfTRPSRYJ7AgmePxoSsHXwWk+IIoNzhKwJzJPOOXOjYDodKXP+Gn1NLHShZ3QB6/WodNis6CPWj/ANDdPEDxNKYZ9/tUvX9MHzNcnpGtRgBI8QfrUevBFpBzLQPCT9Kc4bbNgkK7Z56iBStgOjjL/wDgP+0/M0BWOP8ANP8A41INstgewNBpur41FYimYhOUztEad8UAo44+dDlg6bfzSRjbwEdkD/pmjMYYMoJUZI4Qa9/JISDOveRS0Gn9ff4KHoPmK8OOXBHte4fSn6bNuPYE77VxsmyNUj4fCjQjRjNx/efQfSvf6vcf8U+76U//ACjcaIFe/wBObP7BVQlvKa5SedFUKSsiKnAZrsGjqW0TzyJ+lZl+LFqkFuEhIg7JA+FaY9irCfadbHioVQPxExa2dSEJ/UXrBBICe8GO1yiaz7kxrxusedaIrxKSalF2lITbngKjWmWmjTU08bRpTq3tTua9uIAip3ar45D3BEAq1Ggie+t5sUAITlMpgZfCNPdWR9CcAVckRo2COsVMGDOg79K2C3aCEpQNAkADwAgVvxGPls9QWKQqmWN4w1atFx0wNkgaqUrkB9xWX4v0/uXpCCGUcAj2vNe/pFWyau4OQ1oV0SG1QY03+VYirH7iZ693/wCxf1qw9F+mDilKafdKgsAJKtYIPE76j5d9LDaFgR6xZUQeyDvoZkfKanI3qv4WohwcAQR86nkU4RCkzVY6esJWwlKlEdsHTYwDv3a/CrPNUf8AEe8y9SFA5Tmg8M2mnpS69Rfim9TVLurMojJOXmOdRqnigjOCUZhmTwOs8fDzqSF23tmg95+VMMbEtyOBrKd3Xb5OOc2JvoziJ/OIP7VKyQf7DATptuJ05itaaFYb0LeULlkwDCxEgnTjAA339a3NCtq1ntxdRCXSYWZnc06s1dhQ7/l/FJxRXa8I9KPh6RkmOMTzppDS1A7pmm1wsJBUohIGpJ0A8zUptWT9NMYuHXnW0g9Q2ojQAAlOhJJ9ozOk+VLAtR6TWaNC5J4lKSR68ak7DH7Z4hKHQVH9plJPgFRPlWLuFSfaSRx1ke/altPt5vaI8SD8BQeNtvm0kRw41HWNsAonbhrxFVzA+lSUtFD6yvLGVYEnLyOusc6tdncJcSFpMgwQddQdt6Ae5ajMWTK0gRt8TUokVEXiv1h4p8tqaUkhMCOQoDiTThRoSzSMPLSVCiBVJUqkTwbURK9KEDNOEjSqgTSr0EHLv3jTxqu4itSp7a1TpvlSP+1JHvk91FYVkhvXMrtK7gOB5fWiOEDQAeNPDimP4StauJEjnHvptiPRlZBII04fLzq8s99DfInQRoZ8JFLrjnG3PdZG40U9kiCPWjKKYGmtaNi+HNuBWZCSoAhABgyQI1rO30FJIIggwRxBrn759NeOjd1UVG3qtRV1wrCshzr9rh/j/PfTq8wtpw5nEBR/uI1Pnxq+fFc1n35ZuJr8KVJRZlRJlTiiZ5AACKuSbtH93rWfsqUhISgwBsEnTyAqJ6TY2tpooClBS5G50HE/LzrX6c991G9NsfN1cqIVLaCUtgbZRurxJ19OVVsLoQcilmkZ1hol0TsNfSj4pahKgtsQCdhwPd3U1s3CDPDY86mku5BJEq4DlU3rGnHHyaP0NuVKZb6wjMnQk8ANpPExFWlV0gfuHlr8Ky7o9cO5FaxJ1AOg07+PfUx+YfSkKDyuREzG/Dyq5ZYjvnOrF4L6P7hVR/EFSChokggKVPoCD7qYfmnVEdpR75OtMOkiVlrtE6EbnnI407Jh+L11FYU22tRPHeRxpTihERpyoCU5RqfKRHlQFPEmK5rK7b1P4mejYSq6bSlJUkKBVEwniJI2gifKtgS8ga5x5a1lmD3zLYQlasisvtAbSdiRrU+i/wBTkdUtJ2KSVfzW/MyOTyW2rDeuZnDAPKYMHz2p3ZPQnKojTbWquL9Y0zK17zRxfqncid+0d6r0zWS5u20oUtSgAkFR14ASaxd24L7i3XFe0tRCJ0TqeG3nVk6aJd/LKKVrylSAokk6E7E+MVUMIsUwrjvUdNvDNp/cXJShRGogxpIPiOFVUHSrCLRKZg6fekVDYk2EmRUTrVeTxZ7Ew50Z0haiGyQFxuEkwTrpW3WDbbaEpSdAlIHHQCAdBWCJXGo+xWg9F8VU62ltKyVpTGWdYGx310q4wq/PvApOvgdd/TwqKQglUkTGpmm2S4mNfd8TQS89OoV6Gr2JWIOjTf0Px2oTjw7/AEP0qvvtvk7L07jTR5a44iBqCTvz12peh7Wcuif4M/CkKuU99VVu8O3vB19a865ZPtHXhrR6Htak3Q7/AEpwl/uV/wCJqp9W5MQrn60dtsxt/wCw+tPYWLo5m2J14aRCeA8aC5v4UW5XEnkCaZ2glIJPtVahyqBTK5UAnXUgbcYnu2FOXdoIn0+FM7lUgyJ7vL/Sp6VBrO6DmuQJUmZ5TxgcNuPdVN6W2oDoVEZhr4j+PhU7dpAWlQOXNxGkKHdyP3zph0guUqQEuEIUP3EGJBGvnNZWLly6sPRC7DzCQtKSpIA1A2GlTlzaBQgQNUnbTQj3RVN6HXCUrypM6xO0g6THp6VdMRvEtNLdV7KEqUfITFXL6Y9faLxq9Yt0FbmVA4aCSeQA1JrE8dxQvvKXzOnJKeApON4u7cuqccVJMwOCR/akcBTHLFSI4CizSE16TQZ1ZtJJkzpU2h3huOdV1pUGakm3+yTNY37dfiskXnoCyVKcWYyDskEyc+igY5RPrVjxC4ZZhThCEDXvUf7QBqaof4d400064l1SgXSnKTHVgiRqdwTMa6UnphivWvqCT2ESlPInifX4CtuZ6c/kt660vpD0sUtKkMjq2zvHtK1BEn9u2w99Q9t0hWQUPHOk6SdSPGdxUetVNYHA0z34/R3cnXsnThXMqA1oVsoDRW3wP0ob5Ewnas7z7afKZotxdZlTtt6V6xdKSQUkgjYgwR58KaE0tqtGVu+1/wCj/S3MUouYI2C4GneoR7x/NWa4t0JcTp2TBOukTr7qyHrONaH0YxhLjGZ1UdQghZ/wAJSfQEeVNNMfxFx7KhFsj90LWO79gnx18hzqmWy1ceO0fWmGJ3innVOHdSvTkPIADyrnblWgGka1FVxfjVg67TUVE4gc3lQmrlStCaK4dCI4VnfVdF6+URqFffdTi3fU0oKSSI1BGhFN3gUkUVtQ24cK0ntzdTKvuF9OsyQHgFGIzDnwJT9PSrZhF+h4JWlQPMCOydttxpWLlnkafYVirtusLQdRz2I5HmKpLcVo0OxIqIxtI6omNRGvKab9H8aRdJzDsrGik7weYPI1IX6f0iDxMbbfcUEruHNACVJBnn/NLvbQuAZQEFJOaBEjhtvUr1cIybwdDRnOG3I0jRbdqUiQo+fGioJj+KdO6jSARQ2bcACZPnTwkrjLkNkjWaHhqyQNdvpXmOrls01whWkCJMQfXWtP0/w+eJJI1A4q048AOM15c2iVGMvDUkmT5jfWuS4EkpPDYD46DjTlkZtCaihG3+Hjqiobpgx4acfKonEGg42DpOqT4HT4/GrIpEykyQdxM6bVDPsFAUk8uyeYB0PiIH2KMOVXsAc6p8J23A84Kfp5VLfiL0kSbRtlBldxqf8AFCCMw8cwy+Sqg8RT2wocSR5jtD4n0qs40P1lq7zpyznMfeT61O56OzZqOWY2pArlGkTTQIpUCvFKoTqthSkudoTU9VXM9nRGk91Ntd5otwvgKe2VhMLX7ECBxVpw7qjn+ts25Ecg8Kesr0inOJ2ruRLnVZGwYBgA68+Md5qPSuD4VrEdTLg4Vr76ApFLcOxpKVb0F+PUHSubPA0kDekIOtBFOCiNjSglU054U4QTioFLF+pDa0JMB0JSrwCgr5R5mhu0l8dnwg0AlKQNK4orgfd8KUKDIZVBFS71tIBH2IqIIqRw27/Yr/t76z75/W3h6n1TDEm4UPCmKVRUvig7flUYpokwBqTp405PTPyf9qOhya9r02+SQr2huPvevUuJ5CrjJIYJiyrdwLQddik7KTy7vGtMYxVu4YzNniApJ9pJ5GskKEHUCD3U6sMQcYXnQZH7hwUORFAa6pJmvVNk701w3Fm3m0uJJhQ1AGoPEGnibhP9wHcaWB6hmNYpaJjhSF3aefuNDTdJ5x5VRDY8ZbND6PwGydNSR37CiY6P01eE+lN+jR7KNZ0UfCSPp76rr7V+Hr7MLKtNQBHcKOwoedEvYqHcuYOVB158B79ai+qJ7PrhWVQ18YptiuUozDXTnyP8mm1yyQkqCyTMnbjudKEXVONqSY2MRuTTlGK5jOiJ4hST8R86hMZYzALHEZT57Gp7GR+kT3p+IprcWnZKP2qSCnunh5EekVHUXz/FLUmgqMU4vZHduD4imJM6DQUajHs8aTNer0FITU04ctnN56VdLZsZUiACkDU8xyiq1gliVELI0B07zUrc3WTWYj3VHX1jp8dz3S8cu1ZB21GTB0hMRtUCkV5eX6nDJJjgPnSELgjzrTmZPbLvr5UsHQjlt4UOa5xzjy3FJn77jTQIhUn1pJNJa3rlfWmHrR+Ip2TTRvh4/KjOLohBOKk0WQdKbNKk06Skb0QwUnj5Gva5PEUkn78KYLSa4CKQdDXZSSANSYAHMnagCKckyTJpzh4TJ17X7Qff51Jt9GlJQS4TnjRAiAf8jx8vWg21ogNgPJIzjOFcU8AI3B4+dRepnpc5toT8K0WNuOxHn97VCFetO7m+UkFElaTISo7+7eoxSqOajqYc9bRmnJNRueitORVahb+g16tt/qoKkL3HIjZX1/itGLfdWcdCrlJUpP8AvI0niniB31cG3Fzr8avnvE3nUqpnuPdS02+n8H6VEh1YneO40dNwqN1ev81X+k/hZUxiBJbIO5GvKobok7Dq0neJHkY+nCpy4AKfdVespauEkDdLg46nRUfOlfuNPxaMQenQcPaMc+Anj8KiskHaNfL3cadpuJSSUxJPf3felAXHP1qaIK20DpO4j7++FRDJKVQdTME99S9ue7Y/I6ffLhTDFE5XJGgV6TSpxEY0iEKHemPMgimOIugJYkxOZHnoR8DUnivaCddyB6Grh+HphrVMh11LR0nL+k4qe7WNaVP6Yx0gtRnOwC4Pgr7+NQHZmAdeVfRGFWiXLC2cOjzNi6Vg/wBrzagT/kQtoeEnnUZ06uS7bYiwtKCm3t7F1k5RmStZVmObfXKPfSkFusGeI2kT4ipbozZNuOJ6zUakJ0OYjhv5+VX7o/0puGsCuykom3W000SgHsOq7c8z2zHLSrzj2FrbcvnyUZHV4UEBKpWnI80leZMdkGdOetGD6ZE+7klIIkcoiOYjhVVv73Or2uz4++tR/Gq8u0OQ4tgskvBhKILiRkROcRoqdjNXfHAtYUl1oBlpzBlWyy3EuLuEB0BUa6ZQY2zHnUTn2rru2PnVgg8R60UqTOp85A+NbziT12jELh15VupDFniLtkEAFbZQbcS4I9qFR5qojN08C9cMNhb7tpgriglvNmU688h5WUf8sHXgEzwrRGsKUlJ3ImPOPLemtqgrV1aAVrM5UoBUo8YATqdp8q178UGUJtHkoSAn+qKEJHOyK1CB/kTUr05xm5snbS4UWTZNKYUWkQLkuFpYO49mCdzRRrD21DUccxHmNx414tYjUgeJrV73ElrwUJYQ0/bizZbfCCjrba7Cipx5xJGYg6GRxk7a0rokm+GBKNilsr/MP9bny6M9V2ozHectAZI04CdCD50p5Y58PjX0X0hs1OICH221MKewpNtCQVlRfR+YmNQMnPhm4UC2u1P3Nu46lJLWJ31u2QkD9FFrcEJ79Wx5poD54ZSAZKgPEinYWkjQg+Brd2nTa3eFW7BQpi6ViCycoMtLWLhoA8IzpHhVJ6G3yrvHS8+lKloTcFEJAT+kktoMcwPfrTgZsbgBRBIHL6UlatZGux0r6AwZ599CLppKFXj2GBQJSkJU6lYykjQbqNYj0pNx+ZuPzOUP51daExlC4GiY0igajgsc9ppbFyEKSoESCCBI1rfukKB+XRoP/mwfh/zUU+N6UPsMhKC3cX1426FJBlIbcWAOWqRQNZM/fJSmVGeSSfs1Wr6+VcLAGw0AHCtc6W2twcOtmULZTbKZt23EmOuUtTqEIyyJInLOvOl/iikLLCsuXIu6a2icqmsp79Adaw65+3Rz3PU/rKL3Cv0co1IMjxqsPIIMEEEbitDeWAKruOWiVpzgHON9ogcKOOvwvLx+qzNepPupKhXJNauc8s7lTa0rSYKSCK1HBcV61sLAkk7DgeIrJUnWrH0RxPqnglR7K9D3E7H5GlQvisUWn/dgGcsGZnntEV6u+eniO4ARXOpCtTw50RKDG3wo2jFsu0kEjuqt3IUlxBGwPIaTIJ9DtVsxFIjvGoqt3Cpnn9a1KHduezyIPz250l1IOppFi7mTJ47/AAPwoyhI5eNTVEMGNvv02/miXtuFtmNx2geMgfSgJPEev39KcMOcI94mnPoK9cozITH7VoPvipzCekirVq3SlLkfmC5cFIRBZ6pSQ32lAkleU6COyZI4kuLAKBUnRW57/wCagsUaIaAymQdfInnSw/s+/wBqE9ZKGnUo/KvsqSrq8yitS1IgJcI0JSDJESd6ZdL+lds41dNssXKX7lFs04t3qg0lDBJCkhCyrUE7jiNoqIHA8jr4caa49b6BQ8D4VFtivjEXbY2hGGXdkUr619xlaFAJyJDaklQUSqQdDsDVqvvxBtVv3roafi4Vh5RKW5AtVpU5m/U0kA5YJ74qmKwhx0FbQz/3JEBQ7xO4PdxqIdlJIUCkjcEQR5GjReVh/EPErG7fVcWiLlLrylF7r+qCIhOXIG1EjbWalukXThVze27qFXAtGVWqywogSplQK1BCVlJJA0kjXlVJZT1hCUgk91FeT1cgmI3BiRPxpwvi0IdK8LTfP3KGr0Ju2blq6zdRIDxbKS0ErO2VUyeI3ol909RluE2guWipmwt7ZfYSsItnFqcUspXKcyXFJGWSeMTWZO3Q0AII461N4C2XTpttIjfl6UxOduLTb4vZ/wBP/K3aLpbnXu3CVNqaOZxaCiVqeXmVJWomAfGvOnGMWd3DzLVwLjK22pS+o6jIhJBjItS52/ikWli0CDrJO5Mkgd9MMXZBQtbUS37W0KEwZ8PlS+XttfBk3T93pVYosFNoZdTduWbdmuEthghJJLsg5io5jwnQDvqCuMbZcw1mxKVhQvOuWshPV9WpsoIBzZs2s+zHfVfcEk6pJ10kfI0FT2xkAGnrnb9/tGlZcLbLkquLJSAS0OwwWytZh32SlCgIlUkSmhO9IGUPM9WzcdW1cXV05n6nMpx9t5AQ2AuD2nye0UwANSazbor0hJHVaFQHZ11KRuJ7qmnblJJUFAkbmf00fVXv7hvWcvWujx+Gde6Rh/SvqHMKFy07msE3CF5Qg50rQgNhErAJATBmNhEzUD0U6St2V+LpxC1NKL4WlMZwh3NECYkEjjzruktwlSU6jMTKZOquRjlVeuEgp7948N/GtJdjPycTm5F+xDp9ZdS5asN3Ba/IG0YUsN5isknMuFCEwE6gTvpVf6E4vb26nkXKXCzcW7tuvqgnrEZyk50hWmkH1G9Vlp4cvOllwcKcZtae/EqwcW6h5q5QwF2i2CgNlwm2IUAsFUAFQA0nSfZodr+KNgVIdeYuy61c3L7KUBrIetC0ICyVz7ChMDQ8xvk6FSZPl9aMV6U8JfrzpjhrqbF15q7/ADNqm3ByhnqSEOoW7uvMdAqNuE0rG/xHtLxGS6F5Cbp51otpZCgwpCktIJKokEiRB04ms1XQHKmwbVhwS760dpRzDf61JOIkGRvpVQw90pWIMTpViTiaAgFZjU6cdO6seuffp0cd+vaFxmyyGRxNRdSWK33WqBAhI0HPxpqG615lz2w7s30CDThBpKmOVJQqnYlqPRPEQ+z2vaTorvPA+Y981LqVWbdDsQ6u4Sk+y52Fefsn108zWoJa040gtTlwgTJHqKp+HX6HnVNKhJBV1Z5gTp4iplCddOO9M7jC21KBDYSUmcw7JnnIrS80cOatltqII7J1BHPvH3xp6rbx/wBf5r0PBaSP3Df6igJX5fX7+VKmDsT9/f34UlThBmivtxxoNKGk7O59fv1pxe26XUZVcd9ahW7rKdPP7+VPhiOug358Pv5mqlCCveiziZ6tQXuQnYkfAn0qJvisANuIKCRsoQeXGrmFkkGTPCDHlGxp5Dbycq0hQnQHcacDuDU3mVc6Y81cqYcIBOhg94qZugzcjI6AHMvZXsoHlPHwNWLpHhiGAFIXoo+wUyQOJzDgO+s7W+ovECZKtBxIJ0gVld5ac9LLh/RFbcwkgQSVq5DXYa+6rykpGGC5zQ82y9YoI3H5l1oIUORQnX1qM6MMLQnOqQVAQTvETtG1SSr4dULdpJK3Hw5mASltACIMyrNrqeyk8dqeyI6tsxM480teI2rby21si5Qq3ZSpOdKkWylZ3E5M2hDqh2o9jwry8ZVdfmt2i/YWhKlkthJU5cBYMiUEBI9aavYswm9N00y8V51Kd/UtyhQSwWkZEpdJBJS37UaTMVGPYo7c2ZtnG3V3TtolhxUtEFeYkuGFzlEknSdRpVbEZUL+IDmXGAkq0DDAGsgSFTHjlHuqQw/ElHCHW7cIeyM3ybu2BQl0KdcUUPkKEqSlEnTeY1I0rXTrFrd195fU3CLlAtm2s/VhtPVGHCsBZJBBOWO6YouEdL7RmzSFMum8SxdspWkNhlf5lWbMslQX2YA2PHedHir3/wAZGi4uXFtKQ6j9BtOELYWUR+sq4AWEr4nRAPLN31TWT/8A1cf7rO6I/wDFuq/0l6YG4ubd1CrgW7H5QqZUqApTKsy1BtKygkiIJg6cKe4j04wzr7y4YZuwu7trpp3rA1HWuBsNEAOaJGVc7naAaGaZxv8AEC/bs8JeStAXc9cXj1aYOR1CEwI7PZUatSsZvHL3F2WnGkKt27UWynAEtoLiQtRWYM686xXFukLT1rhrKUrCrQPdYSE5VZ3ELGSFSdEncCrUem1i5dYm5cM3Jt75NulKUBoOpDSAFZpcgSdoJ8qA0Loiu6VatKeWy4312I/n1CFJWAt0Dq9NU542Hs1TekGKuOYIAwlt+1FrZNrCFJDlpcoUC4taMuY5uyN+Z2MllgPTuztvyjbbNx1TDl7mBDRWWXyotgfqQVAlEzA0ME8W7vS6w/JdSlh8PuW9pbPwGg11bCpWtMKkrUCqJ4xykgZugUh1fAedOsadY69f5ZLiWCodWHY6wJge0UkiZnY8qZPHSkB2T/P0pblJt0wO+lGrn0AlnSguGjLGhpsamk8BrwqrqTNAGSKImhBVESKcKipFN7nenCRQLzcVV+hBcNkutx/ejXl2hW2hA/uHrWEtKg1sWCPB5htwqEqSJ/6hofeDUyC+lkQuASaBcXEiBpXV1X1faufoF5JSAU6RqeUcqTh94h1OZJnfxB8/Gva6p/TEfHHj8KEWz3z9++va6kA1NRw4/f39lbaoMxPnt98q6uoOHeaNtuB33pvdXwZTmzAE6AROY8o+Yrq6lb6VzNqEculLUVLMqO/IcgO6gHC0ZkuBIzJ1G8V1dXNOrrqnMsxYrYrIQTlhRIUJlQ00nx5CnKcJSCokKlaSjMDISk7wd5Om/Kurqtz2YhcQWbVTbaUk9ZH6igCmSojKIEEgCY76tuEMSS4ko3OhTMc+UH611dRJlK30rH4ldGVKbN22ASkfqQN0cDHNPw8Ky0+wO6urq3jMMK0pksakV1dSoeNI1inRrq6gPGTqT5V4+rUV1dTIwf1M10zHiK6uqTi7YRgSAjM4nMrkScqe7Q6mpy56PMLSAWwIGhT2feImurqxvVd845k+kDd9CVT+m4I5KB+I+lVfFMIdYVlWB3FJkfUeYrq6nx3bcrDy+PmTYYFlQ4UNQI3rq6t8cozaTvR015XVUiaIkHkKb3fOurqKIbpOtTtjjDiEBKTAE+8k/OvK6o1Vf//Z